astroquery/mast/auth.py

```diff
@@ -82,10 +81,8 @@ def login(self, token=None, store_token=False, reenter_token=False):
         if not info["anon"]:
             log.info("MAST API token accepted, welcome {}".format(info["attrib"].get("display_name")))
         else:
-            warn_msg = ("MAST API token invalid!\n"
-                        "To make create a new API token visit to following link: " +
-                        self.AUTH_URL)
-            warnings.warn(warn_msg, AuthenticationWarning)
+            raise LoginError("MAST API token invalid!\n To make create a new API token"
```

Comment:
delete "make"

Reply:
Ooops! I missed that. Thanks @ceb8 , I'll correct that.